F1WM.UnitTests/Controllers/RacesControllerTests.cs

```diff
@@ -173,5 +177,69 @@ public async Task ShouldReturn404IfDriversStandingsAfterRaceNotFound()
 			standingsServiceMock.Verify(s => s.GetDriversStandingsAfterRace(raceId), Times.Once);
 			Assert.IsType<NotFoundResult>(result.Result);
 		}
+
+		[Fact]
+		public async Task ShouldReturnCachedNextRace()
+		{
+			DateTime? requestParam = null;
+			var cacheKey = $"{CacheKeys.NextRace}_{requestParam}"; ;
+
+			cachingServiceMock.Setup(c => c.TryGetCacheValue<NextRaceSummary>(cacheKey)).Returns(new NextRaceSummary());
+
+			var result = await controller.GetNextRace(requestParam);
+
+			racesServiceMock.Verify(s => s.GetNextRace(requestParam), Times.Never);
+			cachingServiceMock.Verify(c => c.TryGetCacheValue<NextRaceSummary>(cacheKey), Times.Once);
+			cachingServiceMock.Verify(c => c.Set(cacheKey, It.IsAny<NextRaceSummary>(), TimeSpan.FromDays(1)), Times.Never);
+			var okResult = Assert.IsType<OkObjectResult>(result.Result);
+		}
+
+		[Fact]
+		public async Task ShouldReturnCachedLastRace()
+		{
+			DateTime? requestParam = null;
+			var cacheKey = $"{CacheKeys.LastRace}_{requestParam}"; ;
+
+			cachingServiceMock.Setup(c => c.TryGetCacheValue<LastRaceSummary>(cacheKey)).Returns(new LastRaceSummary());
+
+			var result = await controller.GetLastRace(requestParam);
+
+			racesServiceMock.Verify(s => s.GetLastRace(requestParam), Times.Never);
+			cachingServiceMock.Verify(c => c.TryGetCacheValue<LastRaceSummary>(cacheKey), Times.Once);
+			cachingServiceMock.Verify(c => c.Set(cacheKey, It.IsAny<LastRaceSummary>(), TimeSpan.FromDays(1)), Times.Never);
```

Comment:
We should make sure that `Set` isn't called with any param, so concrete `TimeSpan` should be changed to fitting `It.IsAny` setup